.github/workflows/cron-job-its.yml

```diff
@@ -82,8 +115,16 @@ jobs:
       - name: Checkout branch
         uses: actions/checkout@v3
 
+      - name: setup java
```

Comment:
Just a note, we're using the on-runner java now to save time.
Example here: https://github.com/implydata/druid/blob/c882bd95ceea3df9d5dc3624cef6a25681f1af39/.github/workflows/unit-and-integration-tests-unified.yml#L63

Reply:
I understand that, I have used this setup only for `security_vulnerabilities` job to take the advantage of GHA powered maven cache and incase of CVE suppression PRs `security_vulnerabilities` job alone runs and still cache maven dependencies across workflows.

Reply:
That makes sense. However, we can do the same by using the same caching parameters the setup java action uses, which just follows this convention: https://github.com/actions/cache/blob/0638051e9af2c23d10bb70fa9beffcad6cff9ce3/examples.md#java---maven
Again, not a big deal, but will save a bit of time downloading and setting up java.